### ISSUE TITLE
[4.0] Password Strength [a11y]

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#31875]
The password strength script attempts to be accessible by using aria-live to announce when the password matches the site requirements.

https://github.com/joomla/joomla-cms/blob/4.0-dev/build/media_source/system/js/fields/passwordstrength.es6.js#L143

However the code is wrong. For aria-live to work the region must be on the page at the beginning and then updated with content by js. You cannot dynamically add the aria-live region.

This should be done similar to the message layout
https://github.com/joomla/joomla-cms/blob/88e72435024a030cb2ee61fb960051f5f6557254/layouts/joomla/system/message.php#L65

Also in this case it should be assertive and not polite as we want it to be announced immediately that the password meets the site requirements.

You can easily test this with the builtin screen reader in OSX or Windows
